CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_shopify_source v0.12.0
+[PR #79](https://github.com/fivetran/dbt_shopify_source/pull/79) introduces the following changes: 
+## 🚨 Breaking Changes 🚨
+- To reduce storage, updated default materialization of staging models to views. A `--full-refresh` will be required if you have previously run these staging models as tables.
+
```

Comment:
Small note to add an under the hood entry that the maintainer PR template has been updated.

Reply:
updated